aten/src/ATen/native/native_functions.yaml

```diff
@@ -1222,6 +1223,7 @@
 - func: clamp_max_(Tensor(a!) self, Scalar max) -> Tensor(a!)
   device_check: NoCheck   # TensorIterator
   variants: function, method
+  structured_delegate: clamp_max.out
   dispatch:
     CompositeExplicitAutograd: clamp_max_
```

Comment:
Nit: you can remove this composite kernel (same for the other 3), since we codegen functional->out= and inplace->out= composite kernels automatically for structured operators.